### DISCUSSION TITLE
algebra relazionale - 6

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#63] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/07f34181-f502-4199-89df-ef545dcf4273)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-05 110813](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/111ce673-502f-4fcb-98b6-83bf0e1b8341)
![Immagine 2024-02-05 110824](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/3a34385d-0d8d-4e59-8d6c-6256e7375215)

**Reply by Carlo-Armato** [reply]
Se faccio (PARTITA ⨝ FORMAZIONE )  ⨝ RETI alla fine non ottengo la stessa cosa di (PARTITA ⨝ FORMAZIONE )  ⨝ (RETI  ⨝ CALCIATORE)?
Alla fine un giocatore se ha segnato deve pefforza far parte della FORMAZIONE no?

**Reply by daromacarlo** [reply]
vero

**Reply by daromacarlo** [reply]
@Carlo-Armato il resto ti esce uguale??

**Reply by Carlo-Armato** [reply]
Il resto si uguale uguale

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/79638284/2f6bd703-89e9-44f3-a2b0-b5c2460e3a96)
